index.js

```diff
@@ -2,35 +2,33 @@
 const got = require('got');
 const registryUrl = require('registry-url');
 
-function get(keyword, level) {
+function get(keyword, options) {
 	if (typeof keyword !== 'string') {
```

Comment:
I wonder if we should check for sanity of the options object?

Right now a user of this module could store any properties in the options object.
Example:

```
npmKeyword.names('gulpplugin', {fozzel: 'schnoozel'}).then(names => {
	console.log(names.length); // -> 10
	console.log(names);
});
```
The list would contain 10 packages, cause `options.size` would be `undefined` 😂 
https://registry.npmjs.org/-/v1/search?text=keywords:gulpplugin&size=undefined

Reply:
When I said it should default to 250, I meant even if you specify the object with other options. That how it always works.

Reply:
Sorry 🤦‍♂️

Reply:
Fixed wit using `Object.asssign`.